stdlib/float.mli

```diff
@@ -217,6 +254,20 @@ external floor : float -> float = "caml_floor_float" "floor"
     equal to [f].
     The result is returned as a float. *)
 
+external nextafter : float -> float -> float
+  = "caml_nextafter_float" "caml_nextafter" [@@unboxed] [@@noalloc]
+(** [nextafter x y] returns the next representable floating-point
+   value following [x] in the direction of [y].  If [y] is less than
+   [x], it returns the largest representable number less than [x].  If
+   [x] equals [y], the function returns [y].  If [x] or [y] is
```

Comment:
Is it on purpose that you don't have "If [y] is greater than [x], ..." (to avoid being too heavy in the text)?

Reply:
It was supposed to be in the previous sentence but I will change the comment to be more explicit.

Reply:
See c2f1a4a64